requirements.txt

```diff
@@ -7,6 +7,7 @@ boto3
 #asn1
 M2Crypto>=0.36
 autopep8==1.3.3
+cachetools<4
```

Comment:
I am wondering if this file is still used or not. If not, maybe we can remove it from v7r2?

Reply:
We use it in LHCbDIRAC’s CI for Python 2 and I’m not particularly eager to break that while it’s stable. I hope it can be left as is until v8.

Reply:
Ah right. Sure, let's leave it.